src/pages/index.js

```diff
@@ -5,18 +5,18 @@ import Section from "../components/Section.js";
 import ModalWithForm from "../components/ModalWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import ModalWithImage from "../components/ModalWithImage.js";
+import ModalConfirm from "../components/ModalWithConfirm.js";
+import Api from "../components/Api.js";
 import {
-  cardListEl,
   profileFormElement,
   addCardFormElement,
+  editAvatarFormElement,
   profileEditButton,
+  profileAvatarButton,
   addNewCardButton,
-  profileName,
-  profileJob,
   nameInput,
   jobInput,
   options,
-  initialCards,
 } from "../components/utils/constants.js";
 
 //VALIDATION INSTANTIATION:
```

Comment:
* Please, delete `this._disableButton();` from line 75 in `FormValidator` because in the 9th sprint you need to wait for the server response and only in `then` disable the button to be able the click it in case there is a server error